intellij-toml/src/test/kotlin/org/toml/TomlTestBase.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.toml
+
+import com.intellij.testFramework.fixtures.BasePlatformTestCase
+import org.intellij.lang.annotations.Language
+
+abstract class TomlTestBase: BasePlatformTestCase() {
+    @Suppress("TestFunctionName")
+    protected fun InlineFile(@Language("TOML") text: String, name: String = "Cargo.toml") {
+        myFixture.configureByText(name, replaceCaretMarker(text))
+    }
+
+    protected fun checkByText(
+        @Language("Rust") before: String,
+        @Language("Rust") after: String,
+        action: () -> Unit
+    ) {
+        InlineFile(before)
+        action()
+        myFixture.checkResult(replaceCaretMarker(after))
+    }
+}
+
+fun replaceCaretMarker(text: String): String = text.replace("/*caret*/", "<caret>")
```

Comment:
Actually, this function doesn't make sense for TOML. `/*caret*/` is valid block comment in Rust language and it was introduced to avoid syntax errors in injected code.
But TOML langugage doesn't have block comments at all so I don't know how to avoid syntax error when we need caret symbol in TOML code

Reply:
Thanks, that makes sense.